tests/autoyast/validate_salt_formula.pm

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Make sure we are the salt formula has set message-of-the-day(motd)
+# Maintainer: ybontakis <ybonatakisz@suse.com>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    #assert_screen("login-console-motd-message") if get_var("SALT_FORMULAS_PATH");
```

Comment:
we can remove this line now, close [this PR for needles](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/-/merge_requests/1359) and remove reference from 1st comment in this PR.

Reply:
sure. apologies. this is result of many tests, VRs.. i will fix